packages/tailwindcss/src/intellisense.ts

```diff
@@ -20,16 +20,18 @@ export type ClassEntry = [string, ClassMetadata]
 const IS_FRACTION = /^\d+\/\d+$/
 
 export function getClassList(design: DesignSystem): ClassEntry[] {
-  let list: ClassItem[] = []
+  let items = new DefaultMap<string, ClassItem>((utility) => ({
+    name: utility,
+    utility,
+    fraction: false,
+    modifiers: [],
+  }))
 
   // Static utilities only work as-is
   for (let utility of design.utilities.keys('static')) {
-    list.push({
-      name: utility,
-      utility,
-      fraction: false,
-      modifiers: [],
-    })
+    let item = items.get(utility)
+    item.fraction = false
+    item.modifiers = []
```

Comment:
These are already the defaults, so I think we can just do this instead:

```suggestion
    items.get(utility)
```

Reply:
This does look a little weird though 😅

Reply:
Yeah… that's why I didn't write it like that :D

Reply:
We need a new method on the `DefaultMap` for this

```ts
items.touch(utility)
```

Or something